linkedlist/LinkedList.js

```diff
@@ -133,23 +224,59 @@ export class LinkedList {
      * @returns {any | null}
      */
     find(value) {
-        // TODO: implement
+        if (this.#headNode === null) {
+            return null;
+        }
+
+        let temp = this.#headNode;
+        let i = 0;
+        while (temp.nextNode !== null) {
+            if (temp.value === value) {
+                return i;
+            }
+            temp = temp.nextNode;
+            i++;
+        }
+        if (temp.value === value) {
+            return i;
+        };
+        return null;
     }
 
     /**
      * Returns a string representation of the list.
      * @returns {string}
      */
     toString() {
```

Comment:
This should return a string, but it does not.

Reply:
Sorry Chef